frontends/ol-search-ui/src/lib/search.tsx

```diff
@@ -0,0 +1,131 @@
+export type LearningResourceSummary = {
+  id: number
+  title: string
+  image_src: string | null
+  platform: string | null | undefined
+  topics: Array<CourseTopic>
```

Comment:
**Suggestion:** Here and elsewhere, `CourseTopic[]` instead of `Array<CourseTopic>`.

Both work, the `CourseTopic[]` notation is more common in TS, I think. (Possibly in early versions of TS only the Array notation was supported.)